File: JJB/1127/짝수_홀수_개수.py
def solution(num_list):
    answer = []
    
    even_num = 0
    odd_num = 0
    
    for num in num_list:
        if num%2==0:
            even_num += 1
        else:
            odd_num += 1
            
    answer.append(even_num)
    answer.append(odd_num)
    
    return answer

File: JJB/Programmers/231127/짝수_홀수_개수.cpp
#include <string>
#include <vector>

using namespace std;

vector<int> solution(vector<int> num_list) 
{
    vector<int> answer;
    
    int even_num = 0;
    int odd_num = 0;
    
    for(const auto &num : num_list)
    {
        if(num%2==0)
            even_num++;
        else
            odd_num++;
    }
    
    answer.push_back(even_num);
    answer.push_back(odd_num);
    
    return answer;
}